code/resources/assets/js/select2.js

```diff
@@ -15,7 +14,7 @@
     factory(require('jquery'));
   } else {
     // Browser globals
-    factory(kQuery);
+    factory(jQuery);
```

Comment:
@ercanozkaya Yes, does that need to be changed here? We also have kodekit.select2.js which uses kQuery ? https://github.com/timble/kodekit/blob/feature/90-select2/code/resources/assets/js/kodekit.select2.js